lua/nvim-treesitter/fold.lua

```diff
@@ -9,6 +9,7 @@ local M = {}
 -- Especially not for every line in the file when `zx` is hit
 local folds_levels = utils.memoize_by_buf_tick(function(bufnr)
   local lang = parsers.get_buf_lang(bufnr)
+  local max_fold_level = api.nvim_win_get_option(0, 'foldnestmax')
```

Comment:
I'm using the current window here, not sure if that's correct

Reply:
Yeah that's correct.